frontend/src/components/widgets/floating-menus/FloatingMenu.vue

```diff
@@ -214,6 +214,16 @@ export default defineComponent({
 		const workspaceBounds = workspace.getBoundingClientRect();
 		const floatingMenuBounds = floatingMenuContent.getBoundingClientRect();
 
+		if (floatingMenuBounds.bottom + this.windowEdgeMargin <= workspaceBounds.bottom) {
+			const { top } = (this.$refs.floatingMenu as HTMLElement).getBoundingClientRect();
+			if (this.type === "Popover") {
+				floatingMenuContent.style.top = `${top + 10}px`;
+				if (this.$refs.tail) (this.$refs.tail as HTMLElement).style.top = `${top}px`;
+			} else {
+				floatingMenuContent.style.top = `${top}px`;
+			}
+		}
+
```

Comment:
Based on reading this real quick, I don't see what this change is doing. What does it solve that wasn't already implemented before? It's also better to let Vue handle the adding/removing of CSS styles with `:style="..."` instead of using a `ref` to forcibly manipulate the DOM.

Reply:
Previously, if you scroll in the properties and then open the colour picker widget, it opened in the wrong place (as below when I clicked on stroke input on the main branch build) This is because we are using `position: fixed` for the menu content ( https://github.com/GraphiteEditor/Graphite/blob/master/frontend/src/components/widgets/floating-menus/FloatingMenu.vue#L48 ). This was a quick fix based upon the dom manipulation below.
![Wrong position](https://user-images.githubusercontent.com/78500760/163703554-5d3f1e77-512f-41d6-ab13-15bdebf55f96.png)

Reply:
Oh, gotcha. That should be fine, but please switch it to not use a `ref` and comment that code, perhaps with a TODO to have a more robust solution in the future.